pgbackrest_backups_retention.py

```diff
@@ -24,33 +24,43 @@
     print("pip install json sys humanize")
     sys.exit(1)
 
-# Retention parameters definition
-incr_backups_retention_time = 7 #days
-full_daily_backup_retention_time = 30 #days
-full_monthly_backup_retention_time = 12 #months
-full_yearly_backup_retention_time = 20 #years
-
 FULL = 'full'
 INCREMENTAL = 'incr'
 
 # Parse command-line arguments
 parser = argparse.ArgumentParser(description='Backup Retention Script')
-parser.add_argument('--stanza', default='db-production-swiss', help='specify the PgBackRest stanza')
-parser.add_argument('--log-file', default='pgbackrest_backups_retention.log', help='specify the log file path')
+parser.add_argument('--stanza', default='db-production-swiss',
+                    help='specify the PgBackRest stanza')
+parser.add_argument('--log-file', default='pgbackrest_backups_retention.log',
+                    help='specify the log file path')
 parser.add_argument('--dry-run', action='store_true', help='run in dry-run mode')
-parser.add_argument('--mode', default='dev_real_database', help='Specify the mode: dev_sample_data, dev_real_data, production')
+parser.add_argument('--mode', default='dev_real_database',
+                    help='Specify the mode: dev_sample_data, dev_real_data, production')
+parser.add_argument('--incr-BU-RT', default=7,
+                    help='Retention time in days for incremental backups.')
+parser.add_argument('--full-daily-BU-RT', default=30,
+                    help='Retention time in days for full daily backups.')
+parser.add_argument('--full-monthly-BU-RT', default=12,
+                    help='Retention time in months for full monthly backups.')
+parser.add_argument('--full-yearly-BU-RT', default=20,
+                    help='Retention time in years for full yearly backups.')
```

Comment:
What does BU-RT mean, please add clarification or refactor the naming here, if it means "backup retention"  just name it retention instead, for exemple: --incr-retention, -full-daily-retention, --full-yearly-retention...

In the help, please add configuration examples to the user with an explanation